code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -564,12 +564,13 @@
 		if(CHARGE_CRUSH)
 			Paralyze(CHARGE_SPEED(charge_datum) * 2 SECONDS)
 		if(CHARGE_BULL_HEADBUTT)
-			Paralyze(CHARGE_SPEED(charge_datum) * 2 SECONDS)
+			Paralyze(CHARGE_SPEED(charge_datum) * 1.5 SECONDS)
```

Comment:
This just makes the charge not worth it? What are you gonna do with such a short stun?

Reply:
Goal of headbutt is to fish marines for other xenos to reel in even further with their own abilities. 

I think this is the most balanced stun time I can give it. 2 seconds makes it too easy to infinitely chain stun back to back, giving no chance for the marine to fight back if they're in a 1 tile hallway (assuming you time the charges correctly).

1.5 seconds is just barely enough time to land a second charge from 6 tiles away, according to my testing. And since the second charge isn't full speed, the stun will be reduced and make it impossible to chain another charge.

Reply:
If you somehow manage to find a 1 tile hallway long enough for that to be a realistic scenario I'll give you a million dollars.

Reply:
> If you somehow manage to find a 1 tile hallway long enough for that to be a realistic scenario I'll give you a million dollars.

Well... Shipside maints. And also 3x3 mazes if the marine gets unlucky and ends up flung right in the middle of the 3x3. Or just Pitbull mazes that have 1 tile charging lanes built especially for charging/boiler, but those are rare. 
Also Bull can theoretically keep juggling the marine back forth by swapping harm and help intent. Requires skill and timing, but not so much that its unrealistic.